game/audio.cc

```diff
@@ -151,9 +151,9 @@ bool Music::operator()(float* begin, float* end) {
 // 			if (t.pitchFactor != 0) { // Pitch shift
 // 				Buffer tempbuf(end - begin);
 // 				// Get audio to temp buffer
-// 				if (t.mpeg.audioQueue(&*tempbuf.begin(), &*tempbuf.end(), m_pos, t.fadeLevel)) eof = false;
+// 				if (t.mpeg.audioQueue(tempbuf.data(), tempbuf.data() + tempbuf.size(), m_pos, t.fadeLevel)) eof = false;
```

Comment:
this is commented code, isn't it?

Reply:
Correct. Just did a refactor it also took in account these lines it seems. Should work nontheless